RxCocoa/Common/RxCocoa.swift

```diff
@@ -287,6 +287,15 @@ func castOrFatalError<T>(value: AnyObject!) -> T {
     return result
 }
 
+func castOrFatalError<T>(value: Any!) -> T {
+    let maybeResult: T? = value as? T
+    guard let result = maybeResult else {
+        rxFatalError("Failure converting from \(value) to \(T.self)")
+    }
+    
+    return result
+}
+
```

Comment:
I was not sure if you would like to add this, but I've noticed that there was a force unwrap :)
 And yes there is no difference except `Any!`